.gitignore

```diff
@@ -11,6 +11,7 @@
 
 # misc
 .DS_Store
+.env
```

Comment:
Do we need to add some dummy/stock env settings files to the build?

Reply:
Could do - the component throws an error if the variable is unset and tells you how to set it, and the variable is only needed if using that component

Reply:
If the error points you to the solution, I'm good with that. 
Will raise a ticket for documentation. 

Good to go.